design/deployment_design.md

```diff
@@ -0,0 +1,70 @@
+#  **Dockerless CLI -** deployment
+
+This document is the design document for the deployment of the Python CLI. 
+
+Currently, our Python CLI can be deployed in the following ways:
+
+1. Install the CLI via the pip Python package manager like so: `pip3 install conjur-client`. If installed in this way, Python **is** required on machine. If installed via pip, the user can then run the CLI like so `conjur-cli --help`
+
+2. Install and run the executable (Python **is** **not** required on the machine)
+
+   The CLI is delivered as a standalone executable, packed with the required runtime and dependencies. Therefore, there is no need for Python or other prerequisites to be installed on machine. The executable can be fetched by navigating to our [Python CLI release page](https://github.com/cyberark/conjur-api-python3/releases/tag/v0.0.5). If installed in this manner, `./python-cli-executable-location variable get ...`
+
+### Design
+
+#### Pip3 package
+
+We make CLI package available to the public by pushing to a public repository called PyPi. I propose that we maintain pushing our the Python CLI to pip3 for both customer's machines have have internet access and for our development work. The infrastructure is already in-place so this would be at no additional effort.
+
+#### Standalone executable
+
+To install the executable without an internet connection, there are a few options that we can pursue. We can give each executable a different name depending on the intended platform. 
+
+```
+conjur.exe (windows)
+conjur (linux)
+conjur-darwin (macOS)
+```
+
+For Windows, executables have the `.exe` in their name so we will follow suit for the CLI Windows executable. 
+
+For Linux and macOS, we will need to differentiate them. For Linux, we can call the executable `conjur` and for macOS, `conjur-darwin`. That way, the common user-case (Linux) will have a cleaner experience. 
```

Comment:
The requirement is that anywhere you go the cli called conjur. Can we change the release structure to meet this requirement?

Reply:
I agree, they all need to be called conjur. In Windows, yes the exe is required in the end, but the UX remains the same. You can call `conjur` and it works.
If the naming duplication concerns you, I think that the folders approach is the right way.